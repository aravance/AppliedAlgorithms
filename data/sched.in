7
1 3 40
2 1 35
3 1 30
4 3 25
5 1 20
6 3 15
7 2 10
4
3054 2 30
4099 1 35
3059 2 25
2098 1 40
10
29551 8 2861
26324 2 17370
10320 2 26999
15105 1 26326
17591 3 12977
19297 4 26717
25121 5 17402
26788 3 23759
29106 3 24155
15927 2 27595
10
16537 6 1123
2109 3 2655
4258 10 30336
8057 10 32700
10877 10 38457
15895 5 2089
17248 5 6963
18705 1 29379
20122 2 6110
22859 6 2275
10
16537 7 1123
2109 3 2655
4258 10 30336
8057 10 32700
10877 10 38457
15895 5 2089
17248 5 6963
18705 1 29379
20122 2 6110
22859 6 2275
11
16537 7 1123
2109 3 2655
4258 8 30336
23989 10 1122
8057 10 32700
10877 10 38457
15895 5 2089
17248 5 6963
18705 1 29379
20122 2 6110
22859 6 2275
50
275 14 10791
573 19 11653
3247 32 20236
5737 17 31238
6689 49 30070
8976 42 1189
9127 14 20499
9648 41 10216
10320 2 26999
12469 35 28484
12577 7 10756
13424 26 16940
14140 38 17190
14403 9 243
15105 1 26326
15170 26 590
15171 28 22164
15341 41 17893
15927 2 27595
16296 48 12287
16455 26 24018
16658 11 23538
17116 34 2842
17199 35 11760
17391 30 23989
17427 2 23114
17591 3 12977
17922 25 8015
18137 15 24649
19297 4 26717
19568 16 19266
22577 28 18804
22917 1 16778
25116 24 22585
25121 5 17402
26003 31 32755
26324 2 17370
26788 3 23759
27603 33 32290
27622 17 22669
27641 30 21544
27668 21 2916
28112 15 12590
28711 12 271
29106 3 24155
29235 22 1960
29551 8 2861
29791 28 27041
31579 1 11642
32278 18 15132
100
275 14 10791
573 19 11653
1071 37 7504
1695 4 13934
2079 95 15227
2741 25 8590
3072 74 20587
3247 32 20236
3779 16 11317
4067 67 26076
4307 65 4402
5083 55 3789
5737 17 31238
6422 33 3002
6689 49 30070
7382 94 30025
8099 3 4957
8761 95 27623
8976 42 1189
9127 14 20499
9323 20 13942
9523 92 24623
9648 41 10216
10024 11 17536
10136 34 26820
10320 2 26999
12469 35 28484
12577 7 10756
13424 26 16940
14140 38 17190
14403 9 243
14783 45 9005
15105 1 26326
15170 26 590
15171 28 22164
15341 41 17893
15376 16 27046
15927 2 27595
16121 26 3632
16163 66 26829
16296 48 12287
16455 26 24018
16658 11 23538
16855 42 31785
17116 34 2842
17199 35 11760
17364 44 23130
17391 30 23989
17427 2 23114
17591 3 12977
17837 29 8368
17922 25 8015
18137 15 24649
18339 21 23622
18429 42 22934
19297 4 26717
19568 16 19266
19717 96 11091
19943 53 4763
20824 92 9769
21922 23 7746
22577 28 18804
22917 1 16778
23144 17 31412
23460 52 5203
23998 5 9447
24716 2 18131
24962 69 4399
25116 24 22585
25121 5 17402
25751 60 28220
26003 31 32755
26324 2 17370
26379 96 28940
26472 8 30216
26788 3 23759
27196 85 29135
27603 33 32290
27622 17 22669
27641 30 21544
27668 21 2916
28112 15 12590
28167 75 9391
28257 57 2497
28409 91 23865
28711 12 271
29106 3 24155
29235 22 1960
29384 78 9434
29551 8 2861
29743 30 16017
29791 28 27041
30157 22 32528
31187 61 1192
31300 92 1991
31579 1 11642
31797 68 9485
32278 18 15132
32297 2 3491
32568 52 11285
1000
37 46 27786
174 672 28616
213 177 22067
252 122 18525
275 14 10791
302 377 28353
341 398 17796
342 542 27388
382 362 25132
438 437 27790
566 679 25847
573 19 11653
574 297 16148
592 855 6879
599 248 4906
633 139 3061
648 885 22480
659 264 3468
722 519 27615
726 604 24730
802 579 5510
862 769 11294
882 239 18235
894 292 28979
931 688 6058
978 470 24915
1013 435 19766
1020 906 13611
1071 37 7504
1084 796 10181
1140 826 24644
1144 230 32587
1246 634 14367
1342 261 16198
1364 90 50
1394 18 25366
1459 924 19958
1543 445 27290
1649 367 21684
1653 306 8910
1657 299 21331
1695 4 13934
1730 482 29287
1799 646 28757
1871 728 12322
1996 497 7365
2021 927 4831
2048 825 10599
2050 734 16275
2079 95 15227
2089 423 2143
2123 154 27148
2181 479 28672
2189 35 4331
2193 335 805
2296 715 7721
2299 485 28464
2471 719 9759
2477 50 12217
2512 450 6989
2540 972 8739
2549 734 24470
2561 182 26822
2672 310 9075
2705 773 15539
2741 25 8590
2752 883 4393
2778 166 5534
2856 675 12015
2899 212 23588
2944 274 4734
2985 601 22000
3072 74 20587
3092 716 27754
3158 357 29601
3162 220 24842
3238 919 15794
3247 32 20236
3248 78 25119
3277 104 2287
3296 90 12843
3313 681 18776
3317 698 14031
3359 652 18565
3388 998 26994
3437 973 22085
3450 732 24386
3463 694 31956
3569 329 27016
3581 283 8913
3654 284 14011
3691 708 27212
3706 382 17429
3729 708 20728
3730 79 1917
3751 230 8099
3765 162 2416
3779 16 11317
3798 458 4961
3944 335 23303
3954 587 2959
3974 355 6187
3984 23 3066
4010 422 26106
4034 739 13299
4067 67 26076
4085 22 18277
4209 481 25462
4259 123 889
4293 660 23589
4298 898 1843
4307 65 4402
4338 968 13992
4405 77 25659
4428 818 2928
4438 9 12457
4488 571 22513
4583 704 981
4603 701 10727
4615 418 18940
4625 542 13726
4647 502 16483
4651 657 3116
4659 5 3308
4680 756 1737
4734 227 31669
4738 960 20226
4802 839 594
4820 658 10137
4834 404 12113
4900 865 7692
4930 84 4264
5032 31 15638
5060 347 2973
5081 491 27120
5083 55 3789
5187 563 3277
5204 179 15043
5245 384 21346
5290 2 3645
5297 594 27991
5314 286 15477
5367 571 30534
5368 164 278
5438 574 22021
5502 597 8233
5513 250 8909
5515 588 20090
5525 695 93
5540 291 16139
5559 247 20684
5593 516 251
5631 825 1919
5687 62 1993
5715 701 4927
5731 742 1083
5737 17 31238
5779 486 24522
5840 658 12783
5847 856 31127
5868 484 15684
5961 368 25986
5966 685 8607
6000 311 18940
6029 629 3840
6046 558 25912
6047 528 8277
6062 266 5099
6077 850 32231
6160 856 26213
6230 435 3951
6275 331 14120
6281 360 25542
6295 428 8411
6335 718 648
6336 593 6019
6344 932 16841
6373 657 26145
6422 33 3002
6451 656 4380
6461 25 26321
6469 955 20403
6483 137 15942
6498 480 21155
6577 362 30517
6641 484 1816
6652 78 30603
6689 49 30070
6764 474 19438
6859 57 30798
6889 30 7120
6909 357 21447
6914 252 12397
6966 704 24588
7006 95 31818
7009 887 12771
7016 80 5444
7054 236 28127
7139 544 18285
7185 489 31909
7243 573 21787
7286 992 29986
7293 517 23462
7300 431 21162
7329 964 11827
7382 94 30025
7388 464 19145
7401 41 10961
7604 512 25734
7609 352 3739
7618 376 8377
7666 479 14700
7727 641 28542
7774 910 8032
7793 605 22610
7817 180 24064
7899 259 13761
7919 215 8118
7992 535 32722
8033 441 1749
8034 156 14300
8050 235 12196
8099 3 4957
8111 563 4893
8227 184 27328
8260 531 10039
8281 600 24058
8294 263 12299
8301 872 27509
8326 398 25471
8361 441 6240
8431 160 28893
8531 670 9172
8550 913 16250
8605 739 17892
8616 231 1361
8622 692 10420
8633 162 1116
8658 462 27276
8668 58 9610
8696 268 19922
8719 133 20207
8731 580 16956
8761 95 27623
8770 984 4122
8793 172 27214
8840 296 7134
8849 955 29931
8855 627 31479
8863 743 32435
8890 553 19704
8917 398 2262
8940 548 21842
8948 117 12566
8958 509 9466
8976 42 1189
8998 480 10789
9037 407 16636
9076 166 18180
9088 610 32528
9127 14 20499
9143 974 14134
9178 188 7846
9217 493 6096
9220 483 7935
9263 11 24413
9323 20 13942
9344 111 4850
9468 440 11372
9523 92 24623
9528 6 246
9541 432 6543
9542 288 9555
9589 969 1032
9606 628 3558
9635 858 24496
9648 41 10216
9702 421 7109
9708 400 10826
9717 557 26499
9800 685 17797
9833 820 4271
9899 226 4934
9905 28 28921
10024 11 17536
10057 411 8803
10136 34 26820
10225 157 480
10269 674 1261
10293 942 20180
10320 2 26999
10334 976 2080
10360 213 21470
10387 632 25147
10463 71 12324
10487 10 31920
10498 478 6144
10512 695 16008
10574 800 18631
10580 986 2950
10604 764 14261
10627 853 12880
10648 260 24805
10683 784 17870
10774 334 6595
10797 73 32508
10874 616 30269
10891 986 8055
10902 799 17053
10913 891 15469
10915 473 20404
10942 886 7386
11053 514 20263
11055 1 8315
11092 529 1940
11093 465 13831
11149 992 19271
11164 735 4943
11167 302 27612
11189 991 4875
11196 208 20441
11236 820 13912
11271 227 21882
11274 683 16602
11313 939 5375
11347 209 20664
11348 114 23925
11376 855 284
11394 142 13202
11444 364 14523
11445 517 4891
11451 118 18051
11460 895 16087
11523 764 5443
11524 447 6837
11534 499 10373
11555 565 22528
11609 643 1347
11612 983 25506
11622 339 15362
11630 452 10623
11641 290 11304
11717 49 18071
11744 308 27157
11751 968 30618
11791 959 998
11808 619 18561
11842 813 32056
11858 734 21212
11977 159 13087
11985 217 30698
12024 685 175
12060 894 14510
12121 652 29696
12132 770 2613
12159 605 23728
12193 630 28792
12380 54 11546
12384 846 31169
12404 902 19291
12447 333 7892
12469 35 28484
12530 206 1792
12569 333 1108
12573 349 6469
12577 7 10756
12626 309 19015
12633 49 10380
12639 351 17352
12643 573 20032
12651 936 4133
12666 759 25216
12747 854 27123
12749 979 15695
12758 278 14588
12829 370 20917
12841 293 23289
12950 464 32731
12966 780 603
12986 169 19557
13046 414 9470
13106 462 27318
13152 61 25196
13166 643 30550
13193 809 31187
13224 366 31100
13238 693 7797
13253 442 15516
13270 596 9306
13271 625 24165
13321 295 16847
13362 496 5912
13385 806 29814
13409 548 14924
13418 52 27813
13424 26 16940
13427 329 14346
13437 740 1521
13502 508 26684
13663 305 21961
13672 375 18167
13713 381 23841
13730 417 3144
13752 816 13280
13807 85 14357
13827 473 26739
13945 331 31540
13968 667 29394
13996 650 23595
14055 187 8052
14062 397 4983
14066 963 16158
14084 278 24340
14118 52 26457
14140 38 17190
14181 920 11850
14315 176 8577
14323 841 30374
14330 654 7478
14366 153 11892
14397 81 7646
14403 9 243
14414 622 20696
14444 706 25221
14475 662 23071
14480 64 21366
14543 978 29913
14581 877 18712
14600 805 31061
14623 863 12166
14646 54 28396
14664 501 16992
14696 856 27771
14706 680 25208
14726 190 13679
14746 586 29826
14772 515 2319
14777 731 12187
14783 45 9005
14889 601 31871
14905 290 32743
14907 887 25325
15015 199 14061
15040 558 7720
15053 250 10148
15083 464 32692
15105 1 26326
15117 70 30971
15121 134 27901
15170 26 590
15171 28 22164
15183 781 29457
15192 390 6130
15296 842 11465
15303 667 13739
15314 567 9483
15341 41 17893
15368 716 24953
15376 16 27046
15413 462 6141
15455 183 12406
15537 993 23621
15566 468 9294
15580 194 3850
15608 590 9496
15701 249 17624
15735 164 11211
15766 405 9059
15783 230 3749
15866 18 29779
15927 2 27595
15954 738 13830
16020 803 1922
16034 349 8277
16082 593 15038
16109 208 23213
16121 26 3632
16130 881 3169
16147 952 12506
16150 589 4564
16160 562 8908
16163 66 26829
16182 375 15545
16190 278 4883
16240 822 14345
16247 8 13114
16296 48 12287
16306 350 24079
16309 574 28232
16323 702 11673
16355 277 22670
16396 353 21273
16423 856 29838
16451 790 4204
16455 26 24018
16483 655 14629
16587 107 16421
16647 578 8173
16658 11 23538
16682 778 15283
16715 921 24393
16732 713 20260
16781 373 18387
16799 979 26846
16813 232 17382
16848 890 845
16855 42 31785
16893 869 20404
16989 139 7308
17080 987 28631
17116 34 2842
17152 197 4301
17184 286 4342
17199 35 11760
17239 444 26831
17246 565 2942
17294 886 21817
17363 250 5789
17364 44 23130
17391 30 23989
17400 136 24204
17427 2 23114
17434 5 22976
17506 676 4111
17591 3 12977
17678 72 29453
17687 725 17113
17721 122 18652
17729 63 17813
17758 402 11020
17837 29 8368
17838 427 8662
17865 528 18860
17885 723 29579
17896 914 29971
17922 25 8015
17938 668 16732
17941 833 16312
17959 906 11041
18000 655 27147
18066 737 22735
18075 586 30806
18137 15 24649
18163 6 31324
18192 948 14939
18198 910 24799
18211 22 27064
18228 886 26274
18237 342 3154
18296 994 5343
18316 821 28393
18339 21 23622
18429 42 22934
18467 380 29731
18533 15 23600
18541 861 13846
18543 198 2282
18628 794 24841
18663 805 25533
18747 716 22188
18766 96 11965
18767 708 18780
18785 36 10405
18790 82 24901
18816 370 2046
18826 779 22699
18852 928 9476
18903 557 5387
18911 398 27927
18948 208 18642
18989 292 22529
19001 915 24401
19062 365 18672
19090 487 7615
19092 507 25662
19113 482 3387
19215 799 31450
19224 79 24145
19297 4 26717
19333 981 16351
19385 480 16822
19416 248 545
19461 399 14898
19465 432 7832
19486 401 13767
19503 897 27546
19525 315 14693
19532 752 25842
19535 910 6713
19568 16 19266
19604 174 14901
19606 910 16432
19657 935 31200
19666 267 11474
19717 96 11091
19797 422 9157
19828 30 27309
19859 101 1184
19916 900 18182
19943 53 4763
20000 69 21382
20020 405 31651
20047 453 23060
20085 367 16993
20132 355 19186
20156 659 5801
20205 250 18792
20227 207 18166
20248 804 8039
20277 580 8836
20302 298 19035
20310 188 460
20347 360 11721
20455 843 9897
20480 30 23937
20498 644 21339
20566 855 8539
20608 74 18169
20610 179 3489
20666 574 8044
20723 94 4247
20742 666 31492
20784 507 13753
20792 605 12221
20824 92 9769
20832 257 28324
20835 598 5779
20883 434 18499
20893 329 1042
20933 505 10424
20967 65 11157
21010 582 21075
21027 195 20117
21053 413 19155
21088 13 16327
21089 547 17089
21110 531 16656
21140 344 13799
21150 599 10225
21202 154 5703
21245 255 29141
21270 279 1867
21271 178 7050
21489 996 635
21494 108 14249
21609 580 14192
21614 620 15153
21615 524 28938
21643 325 32616
21666 980 19102
21672 397 8176
21725 658 6950
21737 681 2798
21769 360 9591
21791 167 32552
21810 714 4519
21817 474 14590
21856 652 10499
21871 880 6157
21922 23 7746
21969 464 8812
21982 567 11264
22055 28 3454
22057 636 21551
22072 148 6215
22087 847 2685
22105 439 14687
22123 792 15176
22223 88 25871
22251 954 26429
22253 874 28506
22358 380 19239
22418 158 4770
22447 127 4500
22451 311 25063
22461 992 27582
22490 525 10323
22499 43 3197
22532 520 6370
22542 54 29362
22563 264 1914
22577 28 18804
22637 916 5370
22664 937 4295
22728 205 16899
22741 111 31242
22758 656 14815
22798 298 11100
22799 723 10957
22808 42 20214
22904 415 14708
22910 354 27545
22917 1 16778
22968 265 32406
22998 30 27660
23009 766 9509
23064 376 27918
23099 986 2537
23103 308 7112
23129 80 2791
23144 17 31412
23203 124 503
23258 140 14631
23267 403 21097
23287 860 19331
23301 265 8272
23325 420 31237
23330 358 12563
23342 27 605
23424 230 15091
23460 52 5203
23493 442 10765
23541 832 585
23570 26 10396
23588 431 32667
23593 85 4011
23626 866 2733
23634 147 23729
23676 988 27217
23689 228 24624
23787 519 11882
23795 793 32273
23830 523 2688
23899 612 14267
23902 49 13946
23946 695 7604
23998 5 9447
24017 295 12945
24042 386 24039
24064 302 8276
24125 909 24638
24144 702 31545
24146 161 14770
24194 572 30337
24236 735 14931
24255 72 22208
24337 128 14163
24353 679 21506
24364 954 210
24400 24 31885
24479 555 7516
24486 598 16746
24492 808 32667
24557 350 15980
24570 616 30976
24585 635 25093
24631 764 27684
24691 877 11791
24706 627 8052
24709 375 32526
24716 2 18131
24755 980 309
24778 106 14689
24780 608 12807
24819 247 30432
24872 575 7983
24873 235 31892
24904 532 27824
24962 69 4399
24974 826 29226
25055 542 20609
25105 996 32501
25116 24 22585
25121 5 17402
25134 384 17788
25142 12 11781
25182 317 25196
25200 540 8887
25220 878 28974
25262 646 7557
25265 73 1066
25270 709 29520
25314 680 19528
25346 917 9835
25437 468 2989
25477 512 13387
25500 146 30356
25641 978 24221
25675 498 14955
25702 969 29293
25719 61 20533
25738 770 16614
25751 60 28220
25776 573 32296
25781 881 15318
25882 91 21163
25896 768 17570
25919 518 24474
25987 657 21606
26003 31 32755
26118 975 2851
26156 823 4703
26205 451 13534
26206 423 6768
26245 451 29463
26253 11 30693
26281 271 13967
26296 180 15177
26313 333 24051
26324 2 17370
26330 734 7596
26334 215 16875
26356 790 26738
26365 834 12057
26379 96 28940
26386 619 20979
26389 847 31041
26392 333 27139
26397 807 2747
26420 419 23663
26439 607 32415
26472 8 30216
26486 153 24059
26536 319 3717
26555 369 9804
26573 464 31967
26632 706 1652
26738 114 15977
26788 3 23759
26789 63 1153
26845 865 31295
26958 549 11887
26969 356 26012
27025 580 21462
27088 524 12481
27112 30 30478
27196 85 29135
27215 279 4505
27253 161 14783
27296 351 18189
27318 416 19186
27321 175 775
27362 862 25435
27367 371 1591
27391 429 2832
27510 654 22801
27523 622 11224
27603 33 32290
27622 17 22669
27641 30 21544
27668 21 2916
27687 678 18835
27711 261 3630
27816 120 20606
27858 650 11560
27877 94 17672
27878 762 2726
27896 336 5631
27966 523 25209
28007 166 7475
28010 495 15348
28016 397 8470
28030 280 25658
28112 15 12590
28167 75 9391
28194 123 13523
28195 756 806
28232 348 25928
28237 509 28249
28243 105 17814
28257 57 2497
28283 380 5122
28314 161 4552
28327 258 24091
28341 941 29568
28348 27 29222
28356 597 32373
28365 239 23707
28367 226 24230
28398 712 23290
28409 91 23865
28452 138 14315
28495 651 23253
28496 246 1218
28516 109 25833
28581 705 21540
28613 563 11559
28615 537 6239
28619 733 7924
28646 735 11351
28671 438 11148
28711 12 271
28879 451 28865
28880 354 13800
28913 630 14144
29071 834 30196
29103 743 17899
29106 3 24155
29209 361 5715
29217 502 31878
29235 22 1960
29330 556 27134
29365 842 22081
29370 143 21046
29384 78 9434
29393 668 29616
29464 56 4638
29538 24 22882
29551 8 2861
29552 873 23808
29553 342 2043
29578 701 26158
29583 138 28673
29743 30 16017
29766 721 28285
29771 273 25022
29791 28 27041
29813 805 986
29849 553 22626
29853 358 26317
29938 35 17620
29942 942 22832
29944 28 21960
29950 313 23713
29970 662 28034
29978 334 8571
29989 56 29350
30016 803 15617
30085 394 13807
30154 176 2250
30157 22 32528
30162 211 3204
30176 372 18274
30179 637 12300
30202 10 9069
30208 919 20398
30228 107 19044
30270 511 31075
30327 96 6596
30334 80 598
30344 72 32114
30361 805 18487
30419 946 23631
30438 712 31863
30477 141 22132
30486 145 23182
30516 52 3929
30615 399 7841
30632 473 16171
30700 499 9015
30809 772 22627
30852 957 7956
30863 198 17194
30886 734 685
30900 114 936
30972 982 31847
31017 690 2709
31018 474 9829
31050 363 24568
31097 423 20416
31123 18 18048
31187 61 1192
31199 259 7040
31240 650 1555
31272 210 25742
31300 92 1991
31380 561 31293
31383 126 28605
31389 250 16005
31489 722 19350
31511 785 10913
31522 590 16351
31546 414 13629
31579 1 11642
31589 173 13757
31595 56 13862
31603 504 19148
31630 964 30932
31657 440 2788
31681 24 10451
31688 362 20806
31760 714 2654
31797 68 9485
31830 726 19645
31833 873 17753
31841 296 23776
31921 750 30122
31963 635 24545
32057 856 13335
32157 785 32157
32215 771 29370
32225 316 921
32255 238 12664
32268 824 8713
32278 18 15132
32297 2 3491
32404 840 23245
32406 440 8548
32430 428 28736
32485 16 21735
32568 52 11285
32576 825 14571
32616 44 3001
32664 250 12068
32679 733 14139
32689 770 15363
32693 630 31196
32705 66 20926
32736 620 2046
32746 784 30546
0
