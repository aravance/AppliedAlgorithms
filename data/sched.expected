100
70
125729
150964
152087
152087
736288
1367242
15364677
